src/main/java/uk/gov/hmcts/reform/bailcaseapi/domain/service/ShowPreviousApplicationService.java

```diff
@@ -518,6 +526,46 @@ public String getFinancialConditionSupporterLabel(
         return null;
     }
 
+    public String getProbationOffenderManager(
+        BailCase previousBailCase
+    ) {
+        StringBuilder stringBuilder = new StringBuilder("|Probation offender manager||\n|--------|--------|\n");
+        if (previousBailCase.read(HAS_PROBATION_OFFENDER_MANAGER, YesOrNo.class).orElse(YesOrNo.NO) == YesOrNo.YES) {
+            stringBuilder.append("|Probation offender manager|Yes")
+                .append("|\n|Given names|")
```

Comment:
super minor but may as well reuse the givenNames variable here? like a .append("|\n" + givenNames)